algorithm_config.yaml

```diff
@@ -1,8 +1,8 @@
-algorithm_description: Subset GEDI L1B, L2A, L2B, or L4A granules within an area of interest (AOI)
+algorithm_description: Subset GEDI L1B, L2A, L2B, L4A, or L4C granules within an area of interest (AOI)
 algorithm_name: gedi-subset
-algorithm_version: 0.8.0
+algorithm_version: 0.9.0
 repository_url: https://github.com/MAAP-Project/gedi-subsetter.git
-docker_container_url: mas.maap-project.org/root/maap-workspaces/base_images/python:v4.0.0
+docker_container_url: mas.maap-project.org/root/maap-workspaces/custom_images/maap_base:v4.1.0
```

Comment:
Is this the new default registration image? Should ask JPL to update the value in the UI?

Reply:
I don't know if it's the default, nor if it should be. This is the newly added, most basic, stripped down image, which is what we want for GEDI Subsetter, but I don't know how many other algorithms would make use of it.